packages/testing/config/jest/web/index.js

```diff
@@ -1,13 +1,16 @@
 const path = require('path')
-const { TextDecoder } = require('util')
 
-const { getPaths } = require('@redwoodjs/internal')
+const {
+  getPaths,
+  getWebSideDefaultBabelConfig,
+} = require('@redwoodjs/internal')
 
 const rwjsPaths = getPaths()
 const NODE_MODULES_PATH = path.join(rwjsPaths.base, 'node_modules')
 
 module.exports = {
   roots: ['<rootDir>/src/'],
+  testEnvironment: path.join(__dirname, './RedwoodWebJestEnv.js'),
```

Comment:
See comment in web/RedwoodWebJestEnv.js I'd like to revert to `'jest-environment-jsdom'`